chronosphere/resource_notification_policy.go

```diff
@@ -411,14 +415,21 @@ func expandNotificationPolicyRoutes(routes []intschema.NotificationRoute) *confi
 		notifierList.RepeatIntervalSecs = int32(duration.Seconds())
 
 		if r.Severity == "warn" {
+			if out.Warn != nil {
```

Comment:
this is probably unnecessary, but I wanted to be sure we're not letting duplicates through.

This logic needs to be refactored and merged with the independent policy logic, so I'll clean it up as part of that.